.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# pre-commit.ci - ignore updates to codebase made by pre-commit.ci
+5244539071197ce9accfd378bd9f5e557bbc0a49
```

Comment:
No new line at end of file. Not sure if this is needed, but GitHub flagged this for me.

Reply:
GitHub doesn't like it, but I don't think its necessary, Git doesn't seem to mind.